packages/test/service-load-test/src/testConfigFile.ts

```diff
@@ -13,6 +13,7 @@ export interface ITestConfig {
 export interface ITestTenant {
     server: string,
     username: string,
+    driveId: string,
```

Comment:
Why aren't we looking up the drive id at runtime anymore? It's redundant to have to specify it

Reply:
i was thinking the opposite. it never changes, so why look it up every time.

Reply:
The problem is that now to set up, you will have to find out the driverId before you can target a tenant. 
The driveId isn't easy to find manually.

Reply:
my plan was to have a script to generate the config and add the drive id. the problem with discover here is it makes the service load test coupled to odsp which i'm trying to break

Reply:
The test driver should be able to do a rest call and figure it out though.

---
In reply to: [581434492](https://github.com/microsoft/FluidFramework/pull/5230#discussion_r581434492) [](ancestors = 581434492)